### PR TITLE
Fixes in codes and responses

### PR DESCRIPTION
**Pull request by makistsantekidis**

### CODE REVIEW

**Comment by makistsantekidis** [comment]
please review @pkoro @kaggis

File: app/reports/reports_test.go [hunk 1]
```diff
@@ -803,15 +804,28 @@ func (suite *ReportTestSuite) TestWrongUUIDUpdateReport() {
 	// Execute the request in the controller
 	code := response.Code
 	output := response.Body.String()
+	fmt.Println(output)
```
Comment (pkoro) [inline app/reports/reports_test.go]: remove this line

File: doc/v2/docs/reports.md [hunk 2]
```diff
@@ -249,7 +249,7 @@ Accept: application/json
 ```
 
 ## Response
-Headers: `Status: 200 OK`
+Headers: `Status: 200 No Content`
```
Comment (pkoro) [inline doc/v2/docs/reports.md]: Maybe you mean to say "204 No Content"? Same also for L286.

**Comment by pkoro** [comment]
+1

**Pull request merged**